packages/core/src/amazonq/webview/ui/main.ts

```diff
@@ -211,7 +238,29 @@ export const createMynahUI = (ideApi: any, amazonQEnabled: boolean) => {
                 item.formItems !== undefined ||
                 item.buttons !== undefined
             ) {
-                mynahUI.addChatItem(tabID, item)
+                mynahUI.addChatItem(tabID, {
```

Comment:
@ivikash 
Great,
I will adapt this `View diff`/`Accept diff` in `/test` Tab for UTG.

Reply:
We can add UTG to https://github.com/aws/aws-toolkit-vscode/blob/051dd096b65b87ff0f1408515c11ebda0c56da2f/packages/core/src/amazonq/webview/ui/main.ts#L83-L95

Reply:
Yeah I want to add this here but I am trying to add to the  `/test` chat tab.
User can view diff and accept diff directly from code block.